.env.development

```diff
@@ -29,3 +29,5 @@ APPLE_PAY_START_SESSION_URL='http://localhost:18130/payment/cybersource/apple-pa
 APPLE_PAY_AUTHORIZE_URL='http://localhost:18130/payment/cybersource/apple-pay/authorize/',
 APPLE_PAY_SUPPORTED_NETWORKS='amex,discover,visa,masterCard',
 APPLE_PAY_MERCHANT_CAPABILITIES='supports3DS,supportsCredit,supportsDebit',
+STRIPE_PUBLISHABLE_KEY=pk_test_51Li2KoIadiFyUl1xvRRiJohVzLNtnWUYNelHjMkzaf59Mq01ZMdsCGKzh9qyRwCIHBVEv0aQPkrvdH3OJ6F6WjSv00hdOx2EMb
```

Comment:
Will this key have to change in the future to our own test key?

Reply:
This is our own test key from our Zebra POC on the Stripe dashboard: https://dashboard.stripe.com/test/dashboard